identity-android/src/main/java/com/android/identity/SessionEncryption.java

```diff
@@ -130,20 +195,21 @@ void setSendSessionEstablishment(boolean sendSessionEstablishment) {
         mSendSessionEstablishment = sendSessionEstablishment;
     }
```

Comment:
Should we throw `IllegalStateException` if this is called with `mRole == ROLE_MDOC`?

Reply:
We can! I wasn't sure if it needed to be a hard check, but it could be useful for ensuring users know the only ISO-compliant sender of a session establishment is the reader.